File: src/modules/animals.js
const path = require('path');

const loadData = (file) => require(path.join(__dirname, '../../helpers', file));

const animalData = loadData('animal.json');
const catFactData = loadData('catfact.json');
const dogFactData = loadData('dogfact.json');

const getRandom = (arr) => arr[Math.floor(Math.random() * arr.length)];

const random = () => {
    const item = getRandom(animalData.animals);
    return { animal: item };
};

const catFact = () => {
    const item = getRandom(catFactData.facts);
    return { fact: item };
};

const dogFact = () => {
    const item = getRandom(dogFactData.facts);
    return { fact: item };
};

module.exports = {
    random,
    catFact,
    dogFact
};

File: fakedata/modules/animals.py
import random
from ..core import load_data

# Load datasets
animal_data = load_data('animal.json')
cat_fact_data = load_data('catfact.json')
dog_fact_data = load_data('dogfact.json')

def get_random(arr):
    return random.choice(arr)

def random_animal():
    return {"animal": get_random(animal_data['animals'])}

def cat_fact():
    return {"fact": get_random(cat_fact_data['facts'])}

def dog_fact():
    return {"fact": get_random(dog_fact_data['facts'])}
